src/repo/reservoirRepo.js

```diff
@@ -108,23 +108,45 @@ const formatMultipleTimeSeries = (response) => {
   }
 }
 
+const formatReservoir = (reservoir) => {
+  if (!reservoir) { return null }
+
+  // Preferred name is the OSM name if it exists, otherwise the name
+  reservoir.properties.preferred_name = reservoir.properties.osm_name || reservoir.properties.name
```

Comment:
I think it's better to do this here than to have to do it everywhere, but it means that the `preferred_name` property is included in the geojson export, is that ok @SiggyF? Otherwise I can remove it from the export

Reply:
Looks good!